frontend/src/DomplotCSS.tsx

```diff
@@ -1,8 +1,12 @@
-import * as sc from 'styled-components'
+import {css, Global} from '@emotion/core'
+
+import * as React from 'react'
 import * as stripes from './stripes'
 
-export const DomplotCSS = sc.createGlobalStyle`
+const styles = css`
   .striped {
-    background-image: url('data:image/svg+xml;base64,${btoa(stripes.patternSVG)}')
+    background-image: url('data:image/svg+xml;base64,${btoa(stripes.patternSVG)}');
```

Comment:
I'd make the background with gradients to be more readable, but that's mostly personal preference.

Reply:
Cool, tried with things like but they all give some artifacts
```    background: repeating-linear-gradient(
      135deg,
      white 0 2px,
      transparent 2px 4px
    );```
![image](https://user-images.githubusercontent.com/199380/93759819-37d88d00-fc0b-11ea-884c-7720e0d6d4fb.png)